MIGRATION_GUIDE.md

```diff
@@ -364,11 +372,12 @@ The Debug namespace is accessible via `OneSignal.Debug` and provide access to de
 # Limitations
 **General**
 - Recommend using only in development and staging environments for Alpha releases
+- Aliases will be available in a future release
```

Comment:
Update this limitations section to match what we have on the native SDKs.

Reply:
Addressed in [Update Limitations and Notifications permission event listener code sample](https://github.com/OneSignal/react-native-onesignal/pull/1532/commits/25b652c6d55d500956ea8badab6c1540c2c5d221)